pkg/webhook/validate.go

```diff
@@ -106,6 +106,15 @@ func staticValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePol
 			}
 		}
 	}
+
+	// TODO: Add automatic correction in Operator mutation webhook
```

Comment:
Normally, an admission controller doesn't do the automatic correction. I prefer we only set the default value if a flag is not set.

Reply:
removed.

This is related to one idea that we do intelligent analysis, we can discuss it later.